pkg/apis/serving/v1/revision_lifecycle.go

```diff
@@ -173,8 +172,8 @@ func (rs *RevisionStatus) PropagateAutoscalerStatus(ps *autoscalingv1alpha1.PodA
 	// Reflect the PA status in our own.
 	cond := ps.GetCondition(autoscalingv1alpha1.PodAutoscalerConditionReady)
 
-	rs.ActualReplicas = integer.Int32Max(ps.GetActualScale(), 0)
-	rs.DesiredReplicas = integer.Int32Max(ps.GetDesiredScale(), 0)
+	rs.ActualReplicas = ps.ActualScale
+	rs.DesiredReplicas = ps.DesiredScale
```

Comment:
I think we shouldn't pass `-1` down - if these values are `<0` let's just set these to `nil` which will omit this property from being set.